qutebrowser/components/misccommands.py

```diff
@@ -235,12 +236,18 @@ def click_element(tab: apitypes.Tab, filter_: str, value: str, *,
         value: The value to filter for.
         target: How to open the clicked element (normal/tab/tab-bg/window).
         force_event: Force generating a fake click event.
+        select_first: Select first matching element if there are multiple.
     """
-    def single_cb(elem: Optional[apitypes.WebElement]) -> None:
+    def single_cb(elem: Union[apitypes.WebElement, List[apitypes.WebElement], None]) -> None:
```

Comment:
Please add a separate `multiple_cb` callback which is used with the `find_css` handler, which then handles the list and calls `single_cb` with the single element - that should hopefully result in much simpler code.

Reply:
Yes, that's much simpler! Done in 870fd377c323e795b197a62f7406bc6d1a42d6f6.